### DISCUSSION TITLE
5 luglio 2022 - esercizio 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#15] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/aff2a245-95be-44aa-9fde-b37fb7b078fe)

**Comment by niccolo-fato** [comment]
![2024-01-12 22 34 31](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/32ed915b-f71b-4149-aecc-a94968887945)
![2024-01-12 22 34 42](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/45d12f2e-f1f4-4186-bdd8-2b050f18bdac)

**Reply by salicius** [reply]
Alla dipendenza BE->G sulla colonna G andrebbero cambiati i valori con una "a" e non con una "b" o sbaglio ? 
In questo caso avrebbe una riga con tutte "a" e quindi ha un jsp. Correggetemi se sbaglio

**Reply by niccolo-fato** [reply]
Premetto che ho sostenuto questo esame un anno fa, quindi non ricordo perfettamente il funzionamento. Tuttavia, se nelle prime due righe ho **b16** e **b26**, allora per soddisfare la dipendenza devo renderli uguali: quindi o li cambio entrambi in **b16** o entrambi in **b26**. Non posso sostituirli con **a**, perché nessuno dei due lo è

**Reply by salicius** [reply]
Capito, grazie mille della spiegazione, pensavo andassero sostituiti con una a anche se non c'era.

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/e4166393-3188-492c-b292-5b0569a15be6)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/02c50666-ef80-40de-b1dc-fa9bf6819710)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/e87bc630-8263-4afa-b1b9-53012a356492)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/d52aeb10-b7a9-4186-954f-a0c089db6d9e)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/0a9d58b0-0611-4796-9db5-62443b747a77)

**Reply by salicius** [reply]
La tabella è sbagliata sull'attributo B quindi i calcoli andrebbero rifatti credo

**Comment by salicius** [comment]
Io il 2b l'ho svolto in questo modo e mi viene che ρ ha un jsp. (Gia dopo la prima iterazione ha righe con tutte a)

![20250531_164931](https://github.com/user-attachments/assets/abcd551c-6a7b-4998-9030-33c1ff9e1dfe)

**Comment by salicius** [comment]
FYI questo esercizio è uguale a quello di questo altro esame:
https://github.com/sapienzastudentsnetwork/basi-di-dati-1/discussions/27
